lib/assets/javascripts/turbograft/turbolinks.coffee

```diff
@@ -71,14 +71,14 @@ class window.Turbolinks
 
     rememberReferer()
 
-    options.partialReplace ?= false
-    options.onlyKeys ?= []
-    options.onLoadFunction = ->
-      resetScrollPosition() unless options.onlyKeys.length
-      options.callback?()
-
     fetchReplacement url, options
 
+  isPartialReplace = (options, response) ->
```

Comment:
I kind of agree that in the lack of a larger refactor, it feels like this should be in the response (especially since half the logic is already in the response).